packages/webapp/src/routes/developer/email-entry.js

```diff
@@ -20,19 +20,49 @@ const handlers = {
 const getEmailAddressFromPayload = request => {
   const emailAddresses = []
   const err = []
-  const payload = Object.values(request.payload)
 
-  for (const i in payload) {
-    const email = payload[i]
-    const result = emailValidator(email, `#emailAddresses[${i}]`)
-    if (result) {
-      err.push(result.err[0])
+  const fullNames = request.payload.fullNames
+  const emails = request.payload.emails
+
+  if (Array.isArray(emails)) {
+    for (const i in emails) {
```

Comment:
I feel like this loop should be in the processFieldValues, but not necessarily a must

Reply:
I appreciate your suggestion. In my opinion, the same block code is used in both conditions, so I have added this method outside of the loop. Still, I'm open to your advice.
Thanks!